scripts/golinters.bash

```diff
@@ -16,6 +16,11 @@ if [ ! -f "$GOPATH/bin/golangci-lint" ]; then
     exit 1
 fi
 
+SKIP_DIRS=
+if [[ "${CGO_ENABLED}" != "1" ]]; then
+  SKIP_DIRS=--skip-dirs worker/dbaccessor
```

Comment:
Nice, I think eventually we would want to test with `CGO_ENABLED=1`, but that's for another day.